### ISSUE TITLE
CNC Router with Auto-Tool-Changer

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-cad-cam-cnc#69]
# Context

We currently have an industrial Laser Cutter: https://github.com/nelsonic/nelsonic.github.io/issues/455
It's been an awesome intro to CNC and both quieter and _way_ more energy efficient than a CNC Router.
We've used the Laser over the last few months for mini-projects:
e.g: [coat rack](https://github.com/dwyl/learn-cad-cam-cnc/issues/57), [vinyl-record-carry-storage-box](https://github.com/nelsonic/vinyl-record-carry-storage-box/issues/1) and [LuchoTurtle/diy-battery](https://github.com/LuchoTurtle/diy-battery#battery-unit) 

We have come to the painful realisation that we aren't going to laser-cut the [***furniture***](https://github.com/dwyl/home/tree/master/furniture) we need to build for **`@home`** ... 
The maximum plywood material thickness we can reliably cut is **`10mm`** and the burn marks are a pain.
In the short-run we will get around this max thickness constraint by gluing 2 pieces of **`10mm`** together to make **`20mm`** which will be more durable. 
But this doesn't get around the issue of burn marks ... 🔥 
The way we have been solving this in our mini-projects is using a random orbital sander.
But it ends up being a _lot_ of "finishing" work and the end result is still not great.
It's "OK" for workshop, DIY and MVP pieces 
but it's very far from the high-end IKEA/Habitat quality I want for our furniture.

# CNC Router

For the uninitiated, in the world of CNC Routers, watch [April Wilkerson](https://www.youtube.com/c/AprilWilkerson)'s CNC Shop Tour | My CNC Business: https://youtu.be/pn6vUag_mHo

![image](https://user-images.githubusercontent.com/194400/192702226-4d6852ab-e622-4f95-a3f3-1d9207b8add3.png)

And [Jonny Builds](https://www.youtube.com/c/JonnyBuilds)' "This Machine Has Earned Me SO MUCH MONEY": https://youtu.be/QnMkd6YXH4Q
![image](https://user-images.githubusercontent.com/194400/192701443-ad7ddd96-43d3-48ac-b5e5-fa869e0074b0.png)

https://3doutlet.shop/product/cnc-fresadora-1325-c-troca-automatica-e-mesa-de-vacuo/
![image](https://user-images.githubusercontent.com/194400/192696216-9750320c-d630-40e7-879c-ebbb5a57fb37.png)

This is an _incredibly_ good value price for this machine. 
Opening this issue to float the idea of buying one.

# Considerations

+ [ ] ***Where*** to put it?
  It won't fit in our Garage in town. I'm thinking **`New` Workshop** at the **Farm**? 
  https://github.com/dwyl/phase-three/issues/116
  + [ ] ***Security***: this is an _expensive_ machine, it needs to be secured. 
    > Thinking of placing it in in a lockable shipping container 
   https://github.com/dwyl/phase-three/issues/112 
    > _However_ the Gantry is quite wide on the CNC Router so we may need to join a couple of containers together. 
+ [ ] **Power Consumption**: **`5.5KW`** (cutter) + **`7.5KW`** (vac table) + **`3KW`** (extractor) = **`16KW`** ~ **`€3/hour`** 
  > But if we are making high quality furniture and possibly [Tiny Houses](https://github.com/dwyl/tiny-house/issues/24), it will be **10x** worth it! 💰 
+ [ ] **_Consumables_**: the cutting tools will have a useful life and will need to be replaced periodically. This cost will need to be factored into the use of the machine
+ [ ] **Maintenance**: if we install the machine on the [Land](https://github.com/dwyl/phase-three/issues/116) it will be quite far from the tech support (based in Braga). So we will need to be _meticulous_ with our maintenance. e.g: Log books, Videos, etc. 
+ [ ] ***Operator***: initially we would be the ones "feeding" and running the machine, but if we are able to make it a business, we would need to train someone competent to do this. **`#JobCreation`**

**Comment by nelsonic** [comment]
There is a "cheaper" version of this without the tool changer:
https://3doutlet.shop/product/fresadora-cnc-1300x2500x200mm-c-spindle-de-5-5kw/

![image](https://user-images.githubusercontent.com/194400/192742702-a0c1082a-d6ea-4288-b80b-d9ff225f1c00.png)

But the point of the tool changer is _finishing_. i.e. finishing the _edges_ of the material 
such that the input is a blank sheet of material and the output is near-ready to assemble flatpack furniture.
We don't want to have to hand-route each piece to create a rounded edge because this creates a _lot_ of manual work.
If we were to pay a craftsperson **`€1,200/month`** to do "finishing" it would _rapidly_ outweigh the additional cost of having a tool-changing machine.

Basically, we would do the main "cut" with the cutting bit and then do a second pass with the machine to do the round-over. 

Obviously none of this would matter if we are making panels/furniture that doesn't have rounded or beveled edges.
